Towny/src/main/java/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -3150,16 +3152,48 @@ public enum ConfigNodes {
 			"27",
 			"",
 			"# The width of the map shown in /towny map and /res toggle map.",
-			"# Minimum 7, maximum 27, only odd numbers are accepted.");
+			"# Minimum 7, maximum 27, only odd numbers are accepted."),
+	CUSTOM_LISTS("custom_lists", "", "",
+		"# This section of the config allows you to specify custom lists of blocks or entities, that can then be used elsewhere in the config.",
+		"# A custom syntax is used to specify whether to include or exclude certain patterns from the resulting set.",
+		"#",
+		"# Format:",
+		"# *string: Includes all elements that end with 'string'",
+		"# string*: Includes all elements that start with 'string'",
+		"# !*string: Excludes all elements that don't end with 'string'",
+		"# !string*: Excludes all elements that don't start with 'string'",
+		"# #tag: Includes all elements that are contained in the given tag, valid tags can be found here: https://github.com/misode/mcmeta/tree/data/data/minecraft/tags in the blocks or entity_type folders.",
+		"# !#tag: Excludes all elements that are contained in the given tag",
+		"# +string: Includes the element with name 'string'",
+		"# -string: Excludes the element with name 'string'",
+		"# ~string: Includes all elements that contain 'string' in the name",
+		"# !~string: Excludes all elements that contain 'string' in the name",
+		"# c:class: Includes all elements that are instances of the given class"),

```

Comment:
This ought to be boxed in with a header like the other config sections, if the header is a bit wider than they are normally that's fine.